src/ucp/api/ucp.h

```diff
@@ -1906,6 +1906,32 @@ ucs_status_t ucp_ep_create(ucp_worker_h worker, const ucp_ep_params_t *params,
 ucs_status_ptr_t ucp_ep_close_nb(ucp_ep_h ep, unsigned mode);
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ *
+ * @brief Non-blocking @ref ucp_ep_h "endpoint" closure.
+ *
+ * This routine releases the @ref ucp_ep_h "endpoint". The endpoint closure
+ * process depends on the selected @a mode.
+ *
+ * @param [in]  ep      Handle to the endpoint to close.
+ * @param [in]  mode    One from @ref ucp_ep_close_mode value.
+ * @param [in]  param   Operation parameters, see @ref ucp_request_param_t
```

Comment:
what parameters are relevant to this func? Will we be able to specify the cb for this?

Reply:
this is API proposal only, not behavior